packages/libraries/client/src/internal/reporting.ts

```diff
@@ -19,8 +20,20 @@ export function createReporting(pluginOptions: HivePluginOptions): SchemaReporte
 
   const token = pluginOptions.token;
   const reportingOptions = pluginOptions.reporting;
-
   const logger = pluginOptions.agent?.logger ?? console;
+
+  warnIf(
```

Comment:
Since it's required by the `publishSchema()` mutation, shouldn't these be errors?

(I've put errors on the Ruby client for this scenario)

Reply:
It could be an error, yes but not an exception.

Reply:
@charlypoly 93ea93e4635b501031c80b034564f22085f7449f